src/XIOPAK.H

```diff
@@ -114,6 +114,11 @@ uintmax_t FC dskDrvFreeSpace();
 SI     FC xffilcmp( const char * fileName1, const char * fileName2 );
 SEC	xfclear(XFILE* xf);
 
+void getPathRootName(const char* path, char* rootName);
+void getPathRootDirectory(const char* path, char* rootDirectory);
```

Comment:
Is this the "root directory", or just the "directory" of the path?

Reply:
Not quite, it is a weird mix. As an example: For a path as `C:\some\directory\filename.txt`

The output should be `\some\directory\`